app/controller/controllers.py

```diff
@@ -3,15 +3,24 @@
 import json
 import requests
 
-
 get_index_bp = Blueprint('index', __name__)
-get_Sign_Up = Blueprint('Sign_Up',__name__)
+get_Sign_Up = Blueprint('Sign_Up', __name__)
+get_about_page_bp = Blueprint('about_page', __name__)
+
 
 @get_index_bp.route('/')
 def index():
-           # Render index.html
-           return render_template('index.html')
+    # Render index.html
+    return render_template('index.html')
+
+
 @get_Sign_Up.route('/Sign_Up')
 def index():
-          # Render Sign_Up.html
-          return render_template('Sign_Up.html')
\ No newline at end of file
+    # Render Sign_Up.html
```

Comment:
Why are you modifying this line and line 13-14 too ? It doesn't seem to be related the page you are introducing.